### ISSUE TITLE
Broken with domutils v2

### ISSUE THREAD

**Issue by calebcartwright** [crimx/webextension-store-meta#1]
Hi there :wave: We've run into an issue with this package in Shields.io because after updating some other dependencies in our tree (refs https://github.com/badges/shields/pull/6611) `domutils` is now getting resolved to v2. `domutils` is an indirect dependency in our tree, and from what I can tell a direct, but undeclared, dependency here in webextension-store-meta.

Error snippet included below for reference, let me know if you have any questions or need any more information!

```
Error: Cannot find module 'domutils/lib/tagtypes'
Require stack:
- /home/circleci/project/node_modules/webextension-store-meta/lib/dom.js
- /home/circleci/project/node_modules/webextension-store-meta/lib/chrome-web-store/index.js
- /home/circleci/project/services/chrome-web-store/chrome-web-store-base.js
- /home/circleci/project/services/chrome-web-store/chrome-web-store-price.service.js
- /home/circleci/project/core/base-service/loader.js
- /home/circleci/project/core/base-service/loader.spec.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at Function.Module._load (internal/modules/cjs/loader.js:667:27)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (node_modules/webextension-store-meta/lib/dom.js:1:36)
    ```

**Comment by crimx** [comment]
I'll look into it.

**Comment by crimx** [comment]
Fixed with `webextension-store-meta@1.0.4`

**Comment by crimx** [comment]
Should I open a PR on sheilds?

**Comment by calebcartwright** [comment]
> Fixed with webextension-store-meta@1.0.4

Great thanks!

> Should I open a PR on sheilds?

If you're willing to that'd be awesome

**Comment by crimx** [comment]
See PR [#6631](https://github.com/badges/shields/pull/6631).
